spint/tests/test_accessibility.py

```diff
@@ -0,0 +1,27 @@
+"""
+Tests for Accessibility function.
+
+The correctness of the function is veryfied by matching the output to manually calculated values, using very simple dummy dataset.
+
+"""
+
+__author__ = 'Lenka Hasova haska.lenka@gmail.com'
+
+import unittest
+import numpy as np
+from ..flow_accessibility import generate_dummy_flows
+from ..flow_accessibility import Accessibility
+
+
+
+class AccessibilityTest(unittest.TestCase):
+    
+    def test_accessibility(self):
+        flow = generate_dummy_flows()
+        flow = flow.loc[:,['origin_ID', 'destination_ID','distances', 'volume_in_unipartite','dest_masses','results_all=False']]
+        flow['acc_uni'] = function(flow_df = flow, all_destinations=False)
+        
+        self.assertEqual(flow['results_all=False'].all(), flow['acc_uni'].all())
```

Comment:
I don't think this does what you think it does! check what `flow['results_all=False']` looks like! 

You may want to use something like `numpy.testing.assert_allclose` or `numpy.testing.assert_array_equal`, which are designed to compare numpy arrays.

Reply:
I think I addressed this correctly. Let me know if not :).